traefik/docs/counter.traefik_entrypoint_request_duration_seconds_count.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: traefik_entrypoint_request_duration_seconds_count
-brief: How long it took to process the request on an entrypoint, partitioned by status code, protocol, and method.
+brief: Number of requests
 metric_type: cumulative counter
 ---
 ### traefik_entrypoint_request_duration_seconds_count
 
-How long it took to process the request on an entrypoint, partitioned by status code, protocol, and method.
\ No newline at end of file
+How long it took to process the request on an entrypoint, partitioned by status code, protocol, and method.
```

Comment:
Should this say something about number of requests? And somehow duration_seconds doesn't seem to map to number of requests. In any case, this needs to match whatever is in brief:

Reply:
The point of this metric is give the number of "request durations" that have been summed up to give the cumulative traefik_entrypoint_request_duration_seconds_sum metric value